Library/Homebrew/cmd/search.rb

```diff
@@ -127,7 +125,7 @@ def search_tap(user, repo, regex_or_string)
     regex = regex_or_string.is_a?(String) ? /^#{Regexp.escape(regex_or_string)}$/ : regex_or_string
 
     if (HOMEBREW_LIBRARY/"Taps/#{user.downcase}/homebrew-#{repo.downcase}").directory? && \
-       user != "Caskroom"
+       user != "caskroom"
```

Comment:
Out of interest: what difference does the casing make or just aesthetic? I have no preference on this.